src/fsharp/service/service.fsi

```diff
@@ -113,6 +113,9 @@ type public FSharpChecker =
     /// <param name="userOpName">An optional string used for tracing compiler operations associated with this request.</param>
     member ParseFile: filename: string * sourceText: ISourceText * options: FSharpParsingOptions * ?userOpName: string -> Async<FSharpParseFileResults>
 
+    /// Parse a source code file, returning a handle that can be used for obtaining navigation bar information.
```

Comment:
Heh, I see that this summary is similar to the other summary. Would you mind updating both? Something like this:

```
Parse a source code file, returning a type that represents a parsed document whose contents can be traversed for various features.
```

Additionally, a `<param>` comment is needed for each parameter.

Reply:
Sure, I like your summary suggestion.
I'd personally remove param comments for both overloads due to quite self-describing parameter names if it was OK.